Website/src/rendering/src/components/Checkout/PromoInput.tsx

```diff
@@ -29,16 +31,22 @@ const PromoInput = (): JSX.Element => {
   };
 
   const addedPromotions = promotions?.length ? (
-    <ul>
+    <ul className="promotion-list">
       {promotions.map((promotion) => {
         return (
-          <li key={promotion.ID}>
-            <button disabled={loading} onClick={() => handleRemovePromotion(promotion.Code)}>
-              Remove Promotion
+          <li key={promotion.ID} className="promotion">
+            <div>
+              <span className="promotion-code">{promotion.Code}</span>
```

Comment:
When the promotion code is larger than the space available, the promotion amount and remove button are pushed outside the container.

Would it be possible to wrap the code or add ellipsis to it when it is too large?